javagen/src/main/java/com/azure/autorest/util/MethodUtil.java

```diff
@@ -103,6 +102,64 @@ public static Request tryMergeBinaryRequests(List<Request> requests, Operation o
         return selectedRequest;
     }
 
+    /**
+     * @param request   the request to put contentType on
+     * @param operation
+     * @return the created content type parameter
+     */
+    public static Parameter createContentTypeParameter(Request request, Operation operation) {
```

Comment:
Is there any change from the one that delete?

Reply:
no change